great_expectations/core/expectation_configuration.py

```diff
@@ -1327,15 +1351,15 @@ def to_json_dict(self) -> dict:
         return myself
 
     def get_evaluation_parameter_dependencies(self) -> dict:
-        parsed_dependencies = {}
+        parsed_dependencies: dict = {}
```

Comment:
Doesn't `{}` imply `dict`?

Reply:
Yes, it `mypy` wants you to define what kind of `dict`.
I'm sure the intention is for me to say what kind of values the dict contains... but I don't know/was being lazy.